### ISSUE TITLE
[Android] Add multiline support in "Indirect Support" text for non-connected users in Rewards 2.5 panel

### ISSUE THREAD

**Issue by deeppandya** [brave/brave-browser#40931]
https://github.com/brave/internal/issues/1184

**Comment by kjozwiak** [comment]
The above will be verified/checked via https://github.com/brave/brave-browser/issues/40907.

**Comment by kjozwiak** [comment]
Now that https://github.com/brave/brave-core/pull/25588 was merged into `1.70.x`, the above requires `1.70.114` or higher for `1.70.x` verification 👍

**Comment by kjozwiak** [comment]
Verification **PASSED** on `Pixel 6` running `Android 15` using the following build(s):

```
Brave | 1.72.15 Chromium: 129.0.6668.42 (Official Build) canary (64-bit)
--- | ---
Revision | d1172c56951380d9ab0e1f2258bafaa246d84c6a
OS | Android 15; Build/AP41.240823.009; 35; REL
```

Verification notes can be found via https://github.com/brave/internal/issues/1184#issuecomment-2356976158.
